app/models/accounting/quickbooks/full_fetcher.rb

```diff
@@ -16,7 +16,7 @@ def fetch_all
         fetch_qb_data
         restore_accounts!(accounts)
       rescue StandardError => error
-        @qb_connection.destroy
+        @qb_connection&.destroy
```

Comment:
super curious what this change does, if you have a moment to explain.

Reply:
It prevents the crash handler from crashing if there is no quickbooks connection